src/build/env.py

```diff
@@ -13,9 +13,16 @@
 from types import TracebackType
 from typing import Iterable, Optional, Tuple, Type
 
+from packaging.version import Version
```

Comment:
I hadn't caught this. No partial import unless for type hints.
I will fix this and push to your branch to avoid bothering you even more :upside_down_face: